src/components/Components/FieldsTitle.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const FieldsTitle = ({ title }) => (
+  <h3 className="u-title-h2 u-mt-1-half u-mb-1">{title}</h3>
+)
+
+FieldsTitle.propTypes = {
+  title: PropTypes.string
+}
+
+export default FieldsTitle
```

Comment:
@Crash-- Would this be a good candidate to start using `React.memo`? Doesn't have to be in this PR, more asking out of curiosity.

Reply:
It can be, yes!